Core.Collectors/Collector/CollectorBase.cs

```diff
@@ -79,7 +80,12 @@ public async Task<bool> ProcessAsync(T collectionNode, long maxPageCount)
                 counter++;
                 HttpResponseMessage response = await batchingHttpRequest.NextResponseAsync(this.authentication).ConfigureAwait(false);
 
-                if (response.IsSuccessStatusCode)
+                if (response.StatusCode == HttpStatusCode.NoContent)
+                {
+                    // Responses with empty bodies cannot be deserialized
+                    continue;
+                }
+                else if (response.IsSuccessStatusCode)
```

Comment:
If the above is a "break", you don't need the "else" here, right, can you check?

Reply:
correct; no real need for it.